core-ui/src/components/Clusters/views/ClusterOverview/ClusterNodes/ClusterNodes.js

```diff
@@ -1,75 +1,155 @@
 
 import './ClusterNodes.scss';
+import { EMPTY_TEXT_PLACEHOLDER } from 'shared/constants';
+import { StatusBadge } from 'shared/components/StatusBadge/StatusBadge';
 
 const NodeHeader = ({ nodeName }) => {
   const navigateToNodeDetails = nodeName =>
     LuigiClient.linkManager().navigate(`nodes/${nodeName}`);
 
   return (
-    <>
-      <p className="node-header-title">Node:</p>
-      <Link className="fd-link" onClick={() => navigateToNodeDetails(nodeName)}>
-        {nodeName}
-      </Link>
-    </>
+    <Link className="fd-link" onClick={() => navigateToNodeDetails(nodeName)}>
+      {nodeName}
+    </Link>
   );
 };
 
 export function ClusterNodes() {
-  const { nodes, error, loading } = useNodesQuery();
-  const { i18n } = useTranslation();
-  const itemsPerPage = 8;
-  const [currentPage, setCurrentPage] = React.useState(1);
+  const { t, i18n } = useTranslation();
+  const prometheus = useFeature('PROMETHEUS');
+  const usePrometheusQueries = prometheus?.isEnabled;
+
+  const {
+    data: prometheusData,
+    error: prometheusDataError,
+    loading: prometheusDataLoading,
+  } = usePrometheusNodesQuery(!usePrometheusQueries);
+  const {
+    nodes,
+    error: nodesDataError,
+    loading: nodesDataLoading,
+  } = useNodesQuery(usePrometheusQueries === undefined || usePrometheusQueries);
+
+  const data = usePrometheusQueries ? prometheusData : nodes;
+  const error = usePrometheusQueries ? prometheusDataError : nodesDataError;
+  const loading = usePrometheusQueries
+    ? prometheusDataLoading
+    : nodesDataLoading;
+
+  const getStatusType = status => {
+    if (status === 'Ready') return 'success';
+    return undefined;
+  };
 
-  const pagedNodes =
-    nodes?.slice(
-      (currentPage - 1) * itemsPerPage,
-      currentPage * itemsPerPage,
-    ) || [];
+  const getStatus = status => {
+    const conditions = status?.conditions || [];
+    const currentStatus = conditions.find(c => c?.status === 'True');
+    return currentStatus ? (
+      <StatusBadge
+        i18n={i18n}
+        additionalContent={currentStatus.message}
+        resourceKind="nodes"
+        type={getStatusType(currentStatus.type)}
+      >
+        {currentStatus.type}
+      </StatusBadge>
+    ) : (
+      EMPTY_TEXT_PLACEHOLDER
+    );
+  };
+
+  const headerRenderer = () => [
+    t('common.headers.name'),
+    t('cluster-overview.headers.cpu'),
+    t('cluster-overview.headers.memory'),
+    t('common.headers.created'),
+    t('common.headers.version'),
+    t('common.headers.status'),
+  ];
+
+  const rowRenderer = entry => {
+    const { cpu, memory } = entry?.metrics || {};
+    return [
+      <NodeHeader nodeName={entry.metadata?.name} />,
+      cpu ? (
+        <ProgressBar
+          percentage={cpu.percentage}
+          tooltip={{
+            content: t('cluster-overview.tooltips.cpu-used', {
+              percentage: cpu.percentage,
+            }),
+            position: 'bottom',
+            color: 'var(--sapIndicationColor_5)',
```

Comment:
Can we use the same color that we use for in stats-graph CPU: `sapIndicationColor_7`?